### PR TITLE
Added deleteraster command to geomesa-tools 

### PR DESCRIPTION
**Pull request by AndrewAnnex**
to remove raster tables and metadata.

### CODE REVIEW

File: geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/DeleteRasterCommand.scala [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      }
+      try {
+        val queryinfo = s"${table}_queries"
+        if (tableOps.exists(queryinfo)) {
+          logger.info(s"Deleting Query logs table: '$queryinfo'")
+          tableOps.delete(queryinfo)
+        }
+      }
+      try {
+        if (tableOps.exists(GEOMESA_RASTER_BOUNDS_TABLE)) {
```
Comment (jnh5y) [inline geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/DeleteRasterCommand.scala]: A la 'MVC', I'd suggest that little to no business logic by present in the command level.

'Delete(Raster)' should be pushed to the Raster API level and referenced from here.

File: geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/AccumuloRasterStore.scala [hunk 2]
```diff
@@ -79,8 +80,9 @@ class AccumuloRasterStore(val connector: Connector,
   val schema = RasterIndexSchema("")
   lazy val queryPlanner: AccumuloRasterQueryPlanner = new AccumuloRasterQueryPlanner(schema)
 
-  private val tableOps = connector.tableOperations()
+  private val tableOps    = connector.tableOperations()
   private val securityOps = connector.securityOperations
+  private val profileTable  = s"${tableName}_queries"
```
Comment (jnh5y) [inline geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/AccumuloRasterStore.scala]: Nice.

File: geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/AccumuloRasterStore.scala [hunk 3]
```diff
@@ -273,6 +275,41 @@ class AccumuloRasterStore(val connector: Connector,
...
+      }
+    }
+  }
+
+  private def deleteProfileTable(): Unit = {
+    try {
+      if (tableOps.exists(profileTable)) {
+        tableOps.delete(profileTable)
+      }
+    }
```
Comment (jnh5y) [inline geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/AccumuloRasterStore.scala]: I'd suggest factoring out a deleteTable(table) method.  Also, if logging is setup, it should log a note in the case of an exception.

**Comment by jnh5y** [comment]
After the slight refactoring and adding a unit test for the delete function, I'd add a ticket in JIRA and update commit message during merge.

File: geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/AccumuloRasterStore.scala (hunk 3 above)
Comment (AndrewAnnex) [inline geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/AccumuloRasterStore.scala]: logging is not currently setup, should I make the class extend logging or can I raise an exception instead?

File: geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/AccumuloRasterStore.scala [hunk 4]
```diff
@@ -273,6 +275,33 @@ class AccumuloRasterStore(val connector: Connector,
...
+    deleteTable(profileTable)
+    deleteTable(tableName)
+  }
+
+  private def deleteTable(table: String): Unit = {
+    try {
+      if (tableOps.exists(table)) {
+        tableOps.delete(table)
+      }
+    }
```
Comment (jnh5y) [inline geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/AccumuloRasterStore.scala]: I'd extend logging and just log the failure to delete the table.

**Comment by AndrewAnnex** [comment]
merged in: f5f9a04aa19a43194f0aa1b6e232bc64df884f46

**Pull request closed without merging**